.circleci/config.yml

```diff
@@ -401,8 +401,8 @@ workflows:
           name: redshift-itest
           python_version: "3.6"
           db_name: demo-itest
-          requires:
-            - test-3.8
+#          requires:  # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker